code/modules/mob/mob_movement.dm

```diff
@@ -193,14 +193,14 @@
 	if(!isliving(mob))
 		return
 	var/mob/living/L = mob
+	L.setDir(direct)
```

Comment:
Moved setDir here from the lines below because I figure that changing the mob's direction isn't that big of a deal to do early- in fact it would make it more consistent in the event the function returns early.